library/src/skins/default/src/sass/modules/tool/wiki/_wiki.scss

```diff
@@ -490,7 +492,11 @@
       color: var(--sakai-text-color-2);
   }
 
-  .lastmodified, .groupsforthispage { font-size: smaller; text-align: right; color: #555;padding:.5em 0}
+  .lastmodified, .groupsforthispage { 
+      font-size: smaller; 
+      text-align: right; 
+      padding:.5em 0
+   }
   
   .bold-text { font-weight: bold; color:#000 }
```

Comment:
I think we can remove this line. And Use B-5 classes instead.

Reply:
rwiki-tool/tool/src/webapp/WEB-INF/vm/view.vm line 91

Reply:
done